### DISCUSSION TITLE
[회고] 개발자 강연을 듣고 '나'를 탐구하는 시간 갖기

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/challenge-every-week-posting#660] [5. 회고]
노트에 적은 걸 그대로 올리려니 좀 부끄럽네요

[게시글 링크](https://amaran-th.github.io/%EC%A3%BC%EC%A0%80%EB%A6%AC%EC%A3%BC%EC%A0%80%EB%A6%AC/[%ED%9A%8C%EA%B3%A0]%20%EA%B0%9C%EB%B0%9C%EC%9E%90%20%EA%B0%95%EC%97%B0%EC%9D%84%20%EB%93%A3%EA%B3%A0%20'%EB%82%98'%EB%A5%BC%20%ED%83%90%EA%B5%AC%ED%95%98%EB%8A%94%20%EC%8B%9C%EA%B0%84%20%EA%B0%96%EA%B8%B0/)

**Comment by pykido** [comment]
나를 잃어버리지 않아야겠다는 말이 정말 조은데염
